tests/Tests/500__jupyterhub/test-jupyterhub-api-telemetry.robot

```diff
@@ -72,7 +72,7 @@ JupyterHub API Should Exposes SegmentKey
 
 Switch To RHODS Dashboard
     [Documentation]     Switch tab to RHODS dashboard
-    Switch Window    title=Red Hat OpenShift Data Science Dashboard
```

Comment:
should we create a global variable with the Dashboard page title? It is hard coded in more than one/two places I think

Reply:
(not for this PR of course)

Reply:
Wouldn't be a bad idea